eth/eventservices/jobservice.go

```diff
@@ -102,8 +102,14 @@ func (s *JobService) Stop() error {
 }
 
 func (s *JobService) doTranscode(job *lpTypes.Job) (bool, error) {
+	return DoTranscode(s.node, job)
+}
+
+func DoTranscode(node *core.LivepeerNode, job *lpTypes.Job) (bool, error) {
```

Comment:
Since this is exported and callable on its own now should we kill `doTranscode` and directly invoke `DoTranscode` in the callback for `s.eventMonitor.SubscribeNewJob()` in `Start()`?

Reply:
Actually, now that we're going to do all the recovery within ClaimManager, I thought it might be consistent to also handle job recovery completely within JobService (https://github.com/livepeer/go-livepeer/pull/358/commits/76979067fa374cf276425a2d74cb49f671c457b8). So now we don't actually need this particular change anymore.